src/main/java/com/algorand/algosdk/v2/client/algod/GetBlock.java

```diff
@@ -32,6 +32,9 @@ public GetBlock(Client client, Long round) {
     }
 
     protected QueryData getRequestString() {
+        if (!qd.queries.containsKey("round")) {
```

Comment:
round isn't a query parameter, looks like this functionality needs to be extended to distinguish between path/body/query parameters.

Reply:
Thanks for catching this. Yes, this is a regression.